packages/common/src/store/lineup/reducer.ts

```diff
@@ -275,8 +288,16 @@ export const asLineup =
       action.type
     ) as LineupActions<EntryT>['type']
     const matchingReduceFunction = actionsMap[baseActionType]
-    if (!matchingReduceFunction)
+
+    if (matchingReduceFunction) {
```

Comment:
this is a nice fix, just confirming whether we need it for this fix?

Reply:
not for this, but feels like a bug so fixing for the future while we already had the fix